k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -465,6 +469,9 @@ public static void save(StorageEditor editor) {
         editor.putBoolean("hideUserAgent", mHideUserAgent);
         editor.putBoolean("hideTimeZone", mHideTimeZone);
 
+        editor.putString("cryptoProvider", sCryptoProvider);
```

Comment:
I see the S/MIME API being similar but I don't think it needs to be the same app. In fact I don't think it should be. Otherwise every app that uses the API has to support both.

We should have a PGP API and an S/MIME API. They are fundamentally different and so should be different APIs. If OpenKeychain wanted to implement both that would be a decision for that app.

So this should talk about pgpProvider.